src/components/SnippetList.tsx

```diff
@@ -41,8 +44,10 @@ const SnippetList = () => {
               <div className="snippet__preview">
                 <img src={language.icon} alt={language.lang} />
               </div>
-
               <h3 className="snippet__title">{snippet.title}</h3>
+              {query && (
```

Comment:
IMO we could always display the description your opinion on this @dostonnabotov ?

Reply:
I actually had this when I made the search page!
It needs a bit more CSS to make the grid stay pretty though. Or the description needs to be cut off.

Otherwise some cards get bigger than others, causing whitespace underneath smaller cards.

Reply:
I think it's a good idea to do it.

Reply:
Yes, seems like a good idea. Try this to clamp the description to some amount of lines.
```
  .description {
      display: -webkit-box;
      -webkit-box-orient: vertical;
      -webkit-line-clamp: 2;
      overflow: hidden;
      text-overflow: ellipsis;
  }
```

Reply:
Might look weird on smaller screens.
![image](https://github.com/user-attachments/assets/f2afec14-eec5-40af-b443-b45ede31bbff)